var/spack/repos/builtin/packages/pango/package.py

```diff
@@ -43,7 +43,6 @@ class Pango(AutotoolsPackage):
 
     depends_on("pkg-config", type="build")
     depends_on("harfbuzz")
-    depends_on("cairo")
```

Comment:
@adamjstewart and in fact without these changes (i.e. if you just checkout the develop branch), when I run:

spack spec py-pygtk

I get: 

Package cairo requires variant ~X, but spec asked for +X

But then if I run:

spack spec py-pygtk^cairo~X

I get:

Error: Invalid spec: 'cairo~X'. Package cairo requires variant +X, but spec asked for ~X